readme.md

```diff
@@ -70,9 +72,19 @@ Solution consists of 5 logically divided parts (projects):
 Developer should write code for a new module in .Core, .Data and .Web projects as usual in Visual Studio locally on host machine.
 Also developer should write tests for the new module in .Test project.
 
-## How to debug module
+## How to run Virto Commerce Platform Manager
+
+Docker bind Virto Commerce Platform Manager web app to 80 port. So if you have locally installed IIS server you should to stop it or set for local IIS other than 80 port.
+
+Once images are started, open the Virto Commerce Platform Manager - http://localhost . This will launch the application with preinstalled default modules and give you opportunity to configure sample data.
```

Comment:
"Once images are started, open the Virto Commerce" -> "Once the containers are started, open VC"

Reply:
fixed